src/cs/Tests/MonoGame.Extended.Tests/ViewportAdapters/ViewportAdapterTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace MonoGame.Extended.Tests.ViewportAdapters;
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using MonoGame.Extended.ViewportAdapters;
+using MonoGame.Extended.ViewportAdapters.Exceptions;
+using Xunit;
+
+public class ViewportAdapterTests
+{
+    [Fact]
+    public void Constructor_ShouldThrowNullGraphicsDeviceException_IfANullGraphicsDeviceIsPassed()
+    {
+        var act = () => new TestViewportAdapter(null);
+
+        Assert.Throws<NullGraphicsDeviceException>(act);
```

Comment:
After making the suggested change of using `ArgumentNullException` this should be updated to check for that exception type.